src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/Fixtures/storage.expected

```diff
@@ -0,0 +1,20 @@
+open
+validateId
+read
+doRead: 
+read
+doRead: 
+Array
+(
+    [0] => bar
+)
+$_SESSION is not empty
+write
```

Comment:
I thought the lazy_write ini does not call `write` if data didn't change. The data was empty at the start of the session and it is empty at the end (because the flash got added and removed again by reading it). So why does it call write anyway?

Reply:
lazy write calls updateTimestamp instead of write, but only when the session is not empty.
When it's empty, it calls write.